packages/client/hmi-client/src/composables/useCharts.ts

```diff
@@ -187,7 +187,7 @@ export function useCharts(
 	};
 
 	// Create options for forecast charts based on chart settings and model configuration
-	const createForecastChartOptions = (setting: ChartSetting) => {
+	const createForecastChartOptions = (setting: ChartSetting, color: string | null = null) => {
```

Comment:
Looks like you don't need an additional argument, `color` since you can access the color via setting object here. e.g. `setting.primaryColor`